web-registry/src/pages/Dashboard/MyProjects/MyProjects.tsx

```diff
@@ -87,24 +61,47 @@ const MyProjects = (): JSX.Element => {
           <Grid item xs={12} md={6} lg={4}>
             <CreateProjectCard
               isFirstProject={isFirstProject}
-              onClick={submitCreateProject}
+              onClick={() =>
+                submitCreateProject({
+                  createProject,
+                  createWallet,
+                  setError,
+                  navigate,
+                  wallet,
+                  walletData,
+                })
+              }
+              sx={{ height: { xs: '100%' } }}
             />
           </Grid>
         )}
-        {/* TODO: ProjectCards used below temporarily. Will probably be a new variation for this purpose */}
         {isProjectAdmin &&
-          projects?.map((project, i) => (
-            <Grid key={i} item xs={12} md={6} lg={4}>
-              <ProjectCard
-                name={project?.handle || project?.id}
-                imgSrc={''}
-                place="TODO"
-                area={0}
-                areaUnit="ha"
-                track={track}
-              />
-            </Grid>
-          ))}
+          projects?.map((project, i) => {
+            const currentProject = projectsWithOrders.find(
+              projectsWithOrder => projectsWithOrder.id === project?.onChainId,
+            );
+            const name =
+              currentProject?.name ??
+              project?.metadata?.['schema:name'] ??
+              project?.handle ??
+              project?.id;
+            const projectId = project?.onChainId ?? project?.id;
+            return (
+              <Grid key={i} item xs={12} md={6} lg={4}>
+                <WithLoader isLoading={isLoadingProjects} variant="skeleton">
+                  <ProjectCard
+                    {...DEFAULT_PROJECT}
+                    {...(currentProject ?? {})}
+                    name={name}
+                    onButtonClick={() =>
+                      navigate(`/project-pages/${projectId}/edit/basic-info`)
```

Comment:
```suggestion
                      if (project?.onChainId) {
                        navigate(`/project-pages/${project?.onChainId}/edit/basic-info`);
                      else {
                        // TODO: update to navigate to the step the user left off the flow instead of /basic-info 
                        navigate(`/project-pages/${project?.id}/basic-info`);
                      }
```

Reply:
Sorry, I missed that it was a different path for projects without onChainId, it's fixed.

Reply:
This looks good for now. I'll open a task for upgrading this flow to detect where the user left off, not super important right now to do that. @blushi I think we could do a small UI upgrade here, I'll add a task for that.